### PR TITLE
 refacto, chore (egen): refactors code as per template guidelines, hardcodes TF version to 2.15.1,deletes locally generated files

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following changes:

1. Hardcodes TF version to 2.15.1
2. Adds code to delete locally generated files
3. Deletes function to generate UUID and replaces UUID with unique
4. Changes REGION to LOCATION
5. Changes aip to aiplatform
6. Includes Colab restart kernal and authentication as per template guidelines
7. Adds collab enterprise logo
8. Makes structural changes according to the new notebook template
9. Removes the use of future tense
10. Contracts words like it is to it's and do not to don't etc 

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb [hunk 1]
```diff
@@ -1075,13 +985,13 @@
         "id": "serving_function_signature:image"
       },
       "source": [
-        "## Get the serving function signature\n",
+        "#### Get the serving function signature\n",
         "\n",
         "You can get the signatures of your model's input and output layers by reloading the model into memory, and querying it for the signatures corresponding to each layer.\n",
         "\n",
-        "For your purpose, you need the signature of the serving function. Why? Well, when we send our data for prediction as a HTTP request packet, the image data is base64 encoded, and our TF.Keras model takes numpy input. Your serving function will do the conversion from base64 to a numpy array.\n",
+        "For this purpose, you need the signature of the serving function. Why? Well, when we send our data for prediction as a HTTP request packet, the image data is base64 encoded, and our TF.Keras model takes numpy input. Your serving function does the conversion from base64 to a numpy array.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb:992]: - change: "Why? Well, when we send our data for prediction as a HTTP request packet, the image data is base64 encoded, and our TF.Keras model takes numpy input." 
- to: " When the data is sent for prediction as an HTTP request packet, the image data is base64 encoded, and the TF.Keras model takes numpy input.

File: notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb [hunk 2]
```diff
@@ -1157,16 +1067,9 @@
...
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "batchpredictionjobs_create:migration,new,mbsdk"
-      },
-      "source": [
-        "### [predictions.batch-prediction](https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions)"
+        "### Generate batch predictions\n",
+        "\n",
+        "Here is the [Overview of getting predictions on Vertex AI](https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions)"
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb:1072]: This URL involves a redirect. 
- Update from: https://cloud.google.com/vertex-ai/docs/predictions/batch-predictions
- To:  https://cloud.google.com/vertex-ai/docs/predictions/overview

File: notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb [hunk 3]
```diff
@@ -1575,7 +1458,7 @@
         "id": "endpoints_predict:migration,new,mbsdk"
       },
       "source": [
-        "### [predictions.online-prediction-automl](https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl)"
+        "Learn more about how to [Train and use your own models](https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl)"
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb:1461]: The URL involves a redirect.
- update from: https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl
- to: https://cloud.google.com/vertex-ai/docs/training-overview

File: notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb [hunk 4]
```diff
@@ -1673,7 +1556,7 @@
       },
       "outputs": [],
       "source": [
-        "# The format of each instance should conform to the deployed model's prediction input schema.\n",
+        "# The format of each instance should confirm to the deployed model's prediction input schema.\n",
```
Comment (kittyabs) [inline notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb:1559]: change back to "conform" ("confirm" is incorrect")

**Review by kittyabs — COMMENTED** [review]
numerous edits. Also, updated several URLs to avoid redirects

**Review by katiemn — APPROVED** [review]

**Pull request merged**